experiment/resultstore/convert.go

```diff
@@ -124,6 +127,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
+				Value: jobName,
+			},
+			{
+				Key:   "Started Timestamp",
+				Value: strconv.FormatInt(started.Timestamp, 10),
+			},
+			{
+				Key:   "Node",
+				Value: started.Node, // may be empty if node is not specified
+			},
+			{
+				Key:   "Pull",
+				Value: started.Pull, // may be empty if pull value is not specified
+			},
+		},
+	}
+
+	for repo, branch := range started.Repos {
```

Comment:
Ideally it would be nice if we had all the following options for search:
* `org:kubernetes`
* `repo:test-infra`
* `repo:kubernetes/test-infra`
* `branch:master`
* `repo:kubernetes/test-infra:master`
* `repo-version:deadbeef`

https://gcsweb.k8s.io/gcs/kubernetes-jenkins/logs/ci-test-infra-bazel/1214631346354262017/

Reply:
Added all expect repo-version since the mapping field in Testgrid is deprecated.